src/App.jsx

```diff
@@ -55,9 +55,24 @@ function App() {
       timeTable,
       subSlotDict,
     );
-
-    return `/share?v=1&sem=${semID}&data=${compressedBase16URI}`;
-  }
+    const shareableLink = `/share?v=1&sem=${semID}&data=${compressedBase16URI}`;
+    if(navigator.share){
+      navigator.share({
+        title: "VIT-AP TIME TABLE SCHEDULER",
+        text: "Shareable Link",
+        url: shareableLink,
+      })
+      .then(() => {
+        console.log("Link shared Successfully");
+    })
+      .catch((error) => {
+        console.log("Erorr Sharing: ",error);
+      });
+    }
+    else{
+      alert("Your browser doesn't support the Web Share API");
+    }
+  };
```

Comment:
rather than the user getting an alert try to copy the url and alert the user that the url has been coiped to clipboard